lib/github.ts

```diff
@@ -49,7 +49,7 @@ async function fetchIssues(args: {
   perPage?: number;
   page?: number;
 }) {
-  const defaults = { perPage: 100, page: 1, labels: ["devict-help-wanted"] };
+  const defaults = { perPage: 100, page: 1, labels: [] };
```

Comment:
I think leaving the `devict-help-wanted` label in as a default would be good. Since anyone can file issues on any repo, having _something_ that controls what ones show up here and what ones don't seems like a good idea. It means community repos will have to add that tag if they want any issues to show up here unfortunately, but they're still listed in the sidebar. But yeah, keeping this base tag as a requirement feels like the right move.

Reply:
Ok, now that you put it this way that makes total sense, I'm about it 👏  Maybe we can make this clear in the documentation somewhere?